karspexet/ticket/payment.py

```diff
@@ -104,3 +96,30 @@ def _send_mail_to_customer(self):
             [to_address],
             fail_silently=False,
         )
+
+class FakePaymentProcess(PaymentProcess):
+
+    def _charge_card(self):
+        if self.data['payment_success'] == 'true':
+            return True
+        else:
+            raise PaymentError("NO TICKETS FOR YOU")
+
+
+class StripePaymentProcess(PaymentProcess):
+
+    def _charge_card(self):
+        amount = self.reservation.total_price() * 100 # Öre
+        stripe_token = self.data['stripeToken']
+
+        try:
+            return stripe.Charge.create(
+                source=stripe_token,
+                amount=amount,
+                currency="sek",
+                description="Biljetter till Kårspexet"
+            )
+        except (APIConnectionError, AuthenticationError, InvalidRequestError, RateLimitError, StripeError) as e:
+            # Too many requests made to the API too quickly
+            logger.error(e)
```

Comment:
Shouldn't we provide some nice message here? Perhaps provide some context from the current request or similar.